Tests/SessionTests.m

```diff
@@ -50,6 +50,8 @@ - (void)testSessionAutoStartedBackground {
     [mockAmplitude initializeApiKey:apiKey];
     [mockAmplitude flushQueueWithQueue:[mockAmplitude initializerQueue]];
     [mockAmplitude flushQueue];
+    [NSThread sleepForTimeInterval:0.2];
```

Comment:
this is for stableness? Or this would keep failing for you without this?

Reply:
@justin-fiedler Unfortunately tests are failng today